fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt

```diff
@@ -329,10 +329,12 @@
   </xsl:template>
   
   <!-- add utk_mods_accessCondition_local_ms for Local Access Conditions values-->
-  <xsl:template match="mods:mods/mods:accessCondition[@type='local']" mode="utk_MODS">
+  <xsl:template match="mods:mods/mods:accessCondition" mode="utk_MODS">
```

Comment:
@mlhale7 I'm thinking that something like this might be better:
```xml
<xsl:template match="mods:mods/mods:accessCondition[\@type='local' or not(\@type)]" mode="utk_MODS">
```
This will match `accessCondition[@type='local'] OR accessCondition`, but not `accessCondition[@type='whatever']`. 

Or am I completely missing the point here? I'm not sure - it's 3:30 and I'm still not fully caffeinated. :/

Reply:
@CanOfBees - we don't actually have any mods with accessCondition[@type="local"]. This was an incorrect assumption I made initially because of our existing Solr fields (we had this one, but it didn't do anything). In the "local" accessCondition field, I actually want to pull in everything (so I want accessCondition[@type="whatever"]. Currently we still have some MODS with random rights values and accessCondition type attributes (like "local rights statement") and I want these to display somewhere. I plan to use this field only in the click for details section of the item display and not in the facets since the values are not consistent. The accessCondition[@type="use and reproduction"] will be used for facets and accessCondition[@type="restrictions on access"] will display in the click for details section and have a different label. I want the local field to catch everything else.

Reply:
@mlhale7 thanks for clarifying :+1: that helps. This template doesn't need the `xsl:if` conditional; since we have two templates that match on more specific criteria, those have a higher priority for execution by the processor. If you're comfortable dropping the `xsl:if` we can add a note to clarify expected behavior of the template for others. 

My general position here is that we should use (where possible) `xsl:template match="..."` with predicates over using in-template conditionals (but that's just me - I know that we have plenty of things that don't follow that approach).  Let me know if you want to talk about this a bit more.